keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -139,7 +135,7 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
         |application: keel
         |artifacts:
         |- name: keel
-        |  type: DEB
+        |  type: deb
```

Comment:
Is this gonna break anyone's config?

Reply:
Oh, ya maybe probably. I'll hand update their configs.

Reply:
Actually we made these case-insensitive for backwards compatibility.